CesiumGltf/include/CesiumGltf/PropertyView.h

```diff
@@ -108,34 +111,59 @@ class PropertyViewStatus {
    * @brief The property provided an invalid default value.
    */
   static const PropertyViewStatusType ErrorInvalidDefaultValue = 13;
+
+  /**
+   * @brief The property provided an invalid enum value.
+   */
+  static const PropertyViewStatusType ErrorInvalidEnum = 14;
 };
 
 /**
- * @brief Validates a \ref ClassProperty, checking for any type mismatches.
+ * @brief Validates a \ref ClassProperty representing a property, checking for
+ * any type mismatches.
+ *
+ * @tparam T The value type of the PropertyView we are validating.
```

Comment:
I tried not to nitpick this since I doubt this function will be used by anyone else, but... I think it's better to use a neutral voice in the `/** */` documentation

```suggestion
 * @tparam T The value type of the PropertyView that was constructed for this ClassProperty.
```

Reply:
What do you mean by "neutral voice?" If you mean the phrasing using "we," we're using that phrasing, by my count, at over 500 places in our documentation currently.

Reply:
Yes I'm referring to the phrasing using "we", and personally I'd prefer avoiding it. I'm surprised by how much we are apparently using it in our in-code documentation.

It's different if we're talking through logic in a code body or an algorithm, but these are top-level comments 😕

Reply:
I guess I just don't see the issue with using "we"? Not only do we use it in top-level comments in Cesium Native and Cesium for Unreal, but in CesiumJS as well. And my accidental ctrl-shift-f through the whole Cesium for Unreal VS solution says that "we" is used throughout the Unreal source code and even in the MSVC standard library. And I think it makes sense - if you were explaining an algorithm to someone, you say "First we do this step, then we do this second step, and then we take the result and do this to it."

Reply:
I've changed the phrasing for the sake of getting the review done with, but I think if we want to decide on particular dos and don'ts for doc comments, that's something that should be in the style guide.